Update introduction.md

Hun5r4k opened the pull request:
Changed titles/headers to bold for h1 and h2: ## **Title**  Just to read them better.
Added keyword functions, variable functions, arrow functions, return statement. And left some other titles open to work later on. Most changings in functions topic:
I removed the JSDOC-comments (except the first), because it kind of distacts the reader on focusing to learn the syntax.  Mentioning JSDOC-comments once should be enough I think. And yes, I understand the importance of practicing JSDOC-comments when declaring variables, but it doesn't fit in topic functions so much.

Hun5r4k committed:
Update introduction.md

Changed titles/headers to bold for h1 and h2: ## **Title** 
Just to read them better.
Added keyword functions, variable functions, arrow functions, return statement. And left some other titles open to work later on.
Most changings in functions topic:
I removed the JSDOC-comments (except the first), because it kind of distacts the reader on focusing to learn the syntax. 
Mentioning JSDOC-comments once should be enough I think.
And yes, I understand the importance of practicing JSDOC-comments when declaring variables, but it doesn't fit in topic functions so much.

kevinsandow commented:
Making the titles bold should be done in the theme and not in the markdown, the intention with markdown was to move styling content out of the content. There also are two empty sections... nested functions & objects

Hun5r4k committed:
Update introduction.md

- removed bold from h1 and h2 titles which I added last time.
I didn't know about theme using for styling than markdown.
- nested functions & objects done.
I think it should be good for now. Let me know if any topics in introduction.md can be improved/added.
Maybe "control structures" can be splitted into "if's" and "loops", not sure how to, but I'd leave it as it is for now.

Hun5r4k commented:
I send a new Pull request, going to close this.
